data/src/main/java/com/moez/QKSMS/repository/MessageRepositoryImpl.kt

```diff
@@ -299,8 +299,10 @@ class MessageRepositoryImpl @Inject constructor(
         addresses: List<String>,
         body: String,
         attachments: List<Attachment>,
-        delay: Int
+        delay: Int,
+        saveSentMessage: Boolean
```

Comment:
Is there any way to avoid so many API changes? This might cause merge conflicts in the future as we merge upstream QKSMS. Maybe we can pass information in some other way? Like maybe pass a special value in addresses or attachments to indicate if message should or should not be saved? It's a bit hacky, but would save time later.

Reply:
Possible, probably. Feasible, not in my opinion. This place in code is only used a few times. I think if I make changes in multiple methods with the addresses or attachments variable there would be just as much chance for conflicts as it is now, and would be more prone to have other negative side effects.

Reply:
OK, how about moving the new parameter to the top of the methods. That way chance of conflict is lower. And also cleaner diff.

Reply:
I did one now, but the one for `SendMessage.kt` `Params` has more usages where I have to go change them all, so I'm not sure changing that one is a good idea.

Reply:
I see. OK, I think we are good to merge. Thank you for the pull request and fixing my comments.